examples/basicUserStore/user.store.ts

```diff
@@ -0,0 +1,21 @@
+import { Store } from "@leocode/rxstores";
+import { User } from "./user.model";
+
+export class UserStore extends Store<Partial<User>> {
```

Comment:
We can use Loadable model wrapper. We're then extending `Store<Loadable<User>>` and avoid the risk of storing incomplete User model.